client/connect-account-page/strings.js

```diff
@@ -61,7 +61,7 @@ export default {
 
 	terms: createInterpolateElement(
 		__(
-			'By clicking “Finish Setup” you agree to the <a>Terms of Service</a>',
+			'By clicking “Finish Setup”, you agree to the <a>Terms of Service</a>',
```

Comment:
Could the "Finish setup" lowercase be reflected here too?

Reply:
Updated in https://github.com/Automattic/woocommerce-payments/pull/2272/commits/ad664d1738ff13a20b0a8c853d34626537736eb1. Snapshop is also updated in https://github.com/Automattic/woocommerce-payments/pull/2272/commits/533cd0f4d09a3022a97d8db6ec9103e1f4d2c210